tasks/run-collectors/tests/mocks.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -eux
```

Comment:
You should really use the same thing as is used in the task, so that the test is as realistic as possible: `set -xeo pipefail`

Reply:
Nice catch